src/DataProvider/PaginatorInterface.php

```diff
@@ -14,7 +14,8 @@
 namespace ApiPlatform\Core\DataProvider;
 
 /**
- * Paginator Interface.
+ * Paginator Interface. The \Countable implementation should return the number
```

Comment:
We can even remove `Paginator Interface.`. This comments brings nothing.

Reply:
@dunglas done